index.js

```diff
@@ -50,40 +51,90 @@ app.get('/restaurant', (req, res) => {
       })
     }
   }
-  res.json(response)
+  res.json(response);
+
 })
 
 app.post('/restaurant', (req, res) => {
-    if (!req.body.name || !req.body.latitude || !req.body.longitude) {
-        res.json({
-            "code": 404,
-            "status": "Failed",
-            "message": "Missing name, latitude or longitude"
-        })
```

Comment:
A lot of these changes are just formatting (i.e. changing the indentation). Try to avoid these kinds of changes in the same commits as adding or changing the code itself. If you really don't like the formatting, that's fine, but having the changes committed makes it more difficult to see what the actual code changes are.

Reply:
A lot of the times, I just do ctrl+alt+f for the formatting. We should probably always do that before committing+pull requests so that indentation is always the same and these changes wouldn't be recorded.

Reply:
I agree. We need to agree on a standard indentation though. I think we're mostly using 2 spaces, but in some places it's 4 spaces. Not a hard thing to change in our editors though.